client/src/components/Dashboard.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useContext } from "react";
+import {
+  Grid,
+} from "@material-ui/core";
+
+import { MyContext } from "../Provider";
+import API from "../utils/API";
+import { ItemDisplay } from "./ItemDisplay";
+import { AddItem} from './AddItem'
+import {Header} from './Header'
```

Comment:
In situations like this, you should just say `Api` instead of `API`. All caps are usually reserved for project-wide constants.
Also, add spaces around the curly braces in `{Header}`.